page/src/components/ListView/ListView.jsx

```diff
@@ -4,8 +4,6 @@ import {ArrowRight} from 'lucide-react';
 import {filterEvents} from '../../utils';
 import {useCustomContext} from 'app.context';
 
-import EventDisplay from '../EventDisplay/EventDisplay';
```

Comment:
Warning, if you remove this line, list view is no longer running.

Reply:
Not really, EventDisplay is not used anywhere in that file, the IDE was warning me that I left that there accidentally before.